### ISSUE TITLE
Contrast 'Previous result / Next result' too low

### ISSUE THREAD

**Issue by hannolans** [UniversalViewer/universalviewer#460]
The contrast of the sentences 'Previous result' an 'Next result' doesn't have enough contrast with the background for accessibility.

**Comment by edsilv** [comment]
You could make the grey background darker in your theme. This does seem to be quite high contrast to my eye:

![image](https://user-images.githubusercontent.com/1334393/29916628-38e2e442-8e37-11e7-8e17-9684b98358b1.png)

How are you determining this?

**Comment by hannolans** [comment]
I tested the version on http://universalviewer.io/examples to test the version without overriding css.
This issue seems solved as the contrast is enough.
